.github/workflows/build-website.yml

```
@@ -111,5 +109,4 @@ jobs:
             client/test-results/
 
       - name: Build site
-        working-directory: ./client
-        run: npm run build.prod
+  run: npm run build:prod --workspace=@stock-charts/client
```

actionlint (1.7.7)

"steps" section is missing in job "run"

(syntax-check)